streams/src/test/java/org/apache/kafka/streams/processor/internals/ProcessorTopologyTest.java

```diff
@@ -222,9 +222,9 @@ public void shouldDriveGlobalStore() throws Exception {
     }
 
     @Test
-    public void testDrivingSimpleMultiSourceTopology() {
+    public void testDrivingSimpleMultiSourceTopology() throws Exception {
```

Comment:
I guess, this is some "left over" as I cut down this PR to make is smaller... (ie, did remove some other changes that will follow in another RP -- just forgot to undo this changes to -- should be ok to just leave it IMHO) For test in general it's ok to just declare "Exception" as nobody catches it anyway (it's best practice in Java AFAIK).